Packs/TAXIIServer/ReleaseNotes/2_0_33.md

```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### TAXII2 Server
+- Fixed a typo in the *intrusion-set* type output.
```

Comment:
Is it correct that due to this typo, we were creating wrong indicator types? if so let's write this
```suggestion
- Fixed a bug where the integration created wrong indicator types instead of the *intrusion-set* type.
```